js/customizer.js

```diff
@@ -0,0 +1,51 @@
+/* globals wp, UIEvent, jQuery, CodeMirror, _simpleCustomCSSCustomizerExports */
+/* exported simpleCustomCSSCustomizer */
+var simpleCustomCSSCustomizer = ( function( api, $ ) {
+	'use strict';
+
+	var self = {
+		editor: {},
+		textarea: {},
+		control: '',
+		element: ''
+	};
+
+	// Exports must be used to fill the previewData.
+	if ( 'undefined' !== typeof _simpleCustomCSSCustomizerExports ) {
+		$.extend( self, _simpleCustomCSSCustomizerExports );
+	}
+
+	self.init = function() {
+		$( document ).ready( function() {
```

Comment:
Better to do `api.bind( 'ready', function() {` here because it ensures the Customizer is initialized, which happens _after_ DOM ready.